doc/scapy/build_dissect.rst

```diff
@@ -1104,3 +1104,23 @@ Other protocols
     TimeStampField           # NTP (BitField)
 
 
+Design patterns
+===============
+Some patterns are similar to a lot of protocols and thus can be described the same way it Scapy.
+
+The following parts will present several models and conventions that can be followed when implementing a new protocol.
+
+Field naming convention
+-----------------------
+The goal is to keep the writing of packets fluent and intuitive. The recommended way of naming fields is to use
+inverted camel case and common abbreviations. Some examples are on the following list:
```

Comment:
Could you add the following text: "Wherever it is either possible or relevant, Scapy fields names follows the specifications names. This convention helps newcomers to easily forge packets."